src/module/utilities/gtpy_codegenerator.cpp

```diff
@@ -185,113 +196,28 @@ GtpyCodeGenerator::calculatorPyCode(GtCalculator* calc)
 QString
 GtpyCodeGenerator::propValToString(GtAbstractProperty* prop)
 {
-    QString val;
+    QString val{};
 
-    if (qobject_cast<GtModeProperty*>(prop))
```

Comment:
What happened to the modeproperty cast?

Reply:
Since the mode property is of type `GtProperty<QString>` it is handled in line [212](https://github.com/dlr-gtlab/python-module/pull/573/files#diff-0c3f2c7262ecedd19fc140faee42cee251c7d6ab3bd82c4467d48f2ef3506d88R212).